src/main/java/se/liu/ida/hefquin/engine/query/impl/SPARQLQueryImpl.java

```diff
@@ -39,4 +39,9 @@ public Query asJenaQuery() {
 		return jenaQuery;
 	}
 
+	@Override
+	public String toString(){
+		return this.asJenaQuery().getQueryPattern().toString();
```

Comment:
```suggestion
		return this.asJenaQuery().toString();
```
`getQueryPattern()` returns only the pattern of the WHERE clause.